TestSuite/release_notes.py

```diff
@@ -0,0 +1,31 @@
+import json
+
+from pathlib import Path
+from demisto_sdk.commands.update_release_notes.update_rn import UpdateRN
+
+
+class ReleaseNotes(UpdateRN):
```

Comment:
it's odd that ReleaseNoted is inherited from UpdateRN.
Let's get rid of the inheritance add all of the functionality to ReleaseNotes without inheritance.